moved/src/move_execution/tests/transfer.rs

```diff
@@ -111,13 +115,16 @@ fn test_eoa_base_token_transfer() {
     // Should fail when transfer is larger than account balance
     let receiver = ALT_EVM_ADDRESS;
     let transfer_amount = mint_amount.saturating_add(U256::from(1));
-    let outcome = ctx.transfer(receiver, transfer_amount, 0);
-    outcome.vm_outcome.unwrap_err();
+    // Still need to set gas limit for proper functioning of the gas meter
+    let outcome = ctx.transfer(receiver, transfer_amount, 0, 100, U256::ZERO);
+    // Insufficient intrinsic gas
```

Comment:
What is this comment about? In this test it looks like the error should be insufficient balance, not anything about gas.

Reply:
I guess I left it when debugging the broken test and forgot to revert it, good catch!